### PR TITLE
Update to 7.11.0

### PR DESCRIPTION
**Pull request by RhCheGithubBot**
Tracking changes for fixing compatibility with upstream 7.11.0-SNAPSHOT. This PR was created automatically by Jenkins from job devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io

### CODE REVIEW

**Comment by ibuziuk** [comment]
[test]

File: dockerfiles/e2e-saas/docker-entrypoint.sh [hunk 1]
```diff
@@ -143,7 +143,6 @@ export DISPLAY=:1.0
 
 echo "Applying patches..."
 # currently in /tmp/rh-che folder
-patch -u ../e2e/pageobjects/dashboard/Dashboard.ts -i ../test_patches/Dashboard.patch
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/docker-entrypoint.sh]: @Katka92 so, in 7.11 e2e tests are not dependent on the UD tabs?

**Comment by ibuziuk** [comment]
[test]

File: dockerfiles/e2e-saas/docker-entrypoint.sh (hunk 1 above)
Comment (Katka92) [reply dockerfiles/e2e-saas/docker-entrypoint.sh]: They are, but only on Workspaces, Stacks and Getting started.

**Comment by ibuziuk** [comment]
@sleshchenko for `7.11` I'm removing the workaround for 'f?url' parameter - https://github.com/redhat-developer/rh-che/pull/1828/commits/3a6bac92c559dd7ad17fc24d48c16b4575baf257

**Comment by ibuziuk** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
failed to get user token, retesting

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
Failed to get token for a test user. Running tests again.

**Comment by Katka92** [comment]
[test]

File: dockerfiles/e2e-saas/docker-entrypoint.sh [hunk 2]
```diff
@@ -31,6 +31,9 @@ fi
 if [ -z $DEBUG_LEVEL ]; then
   DEBUG_LEVEL="DEBUG"
 fi
+if [ -z $DEBUG_LEVEL ]; then
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/docker-entrypoint.sh:34]: @ScrewTSW this line already exists above

**Pull request merged**